oauth2_http/java/com/google/auth/oauth2/OAuth2Utils.java

```diff
@@ -17,7 +19,7 @@
  * Internal utilities for the com.google.auth.oauth2 namespace.
  */
 class OAuth2Utils {
-  static final String TOKEN_SERVER_URL = "https://accounts.google.com/o/oauth2/token";
+  static final URI TOKEN_SERVER_URI = parseStaticUri("https://accounts.google.com/o/oauth2/token");
```

Comment:
That's not what I was getting at.

s/parseStaticUri/URI.create/

URI.create doesn't throw a checked exception, so is safe from a static field.